features/collections/collectible_multiple_collections.feature

```diff
@@ -11,13 +11,7 @@ Feature: Collectible items in multiple collections
       And I am logged in as a random user
       And I set my preferences to allow collection invitations
       And I post the work "Blabla" to the collection "ModeratedCollection"
-    When I view the work "Blabla"
-      And I fill in "Collection name(s):" with "ModeratedCollection2"
-      And I press "Invite"
-    Then I should see "You have submitted your work to the moderated collection 'ModeratedCollection2'."
-      And I should see "It will not become a part of the collection until it has been approved by a moderator."
-    When I follow "Edit"
-      And I press "Post"
+    When I edit the work "Blabla" to be in the collections "ModeratedCollection,ModeratedCollection2"
```

Comment:
Can you rename this scenario? This still refers to the "Add To Collections" form.